.github/CODEOWNERS

```diff
@@ -32,3 +32,4 @@ src/App/Resources/AppResources.resx
 src/watchOS/bitwarden/bitwarden\ WatchKit\ Extension/Localization/en.lproj
 store/apple/en
 store/google/en
+store/google/Publisher
```

Comment:
🤔 I'd add a different comment section here, as the `Publisher` is not part of the translations, this is used to publish the app in the `Deploy to Play Store` step of the `build.yml`. What do you think about something like:
```suggestion

## Utils ##
store/google/Publisher
```

Reply:
Sure, that works for too.